pkgs/development/python-modules/jupyter-contrib-core/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, jupyter-core
+, notebook
+}:
+
+buildPythonPackage rec {
+  pname = "jupyter-contrib-core";
+  version = "0.4.2";
+
+  src = fetchFromGitHub {
+    owner = "jupyter-contrib";
+    repo = "jupyter_contrib_core";
+    rev = "refs/tags/${version}";
+    hash = "sha256-UTtK+aKxBFkqKuHrt1ox8vdHyFz/9HiKFl7U4UQcG88=";
+  };
+
+  propagatedBuildInputs = [
+    jupyter-core
+    notebook
+  ];
+
```

Comment:
We can't run the tests https://github.com/Jupyter-contrib/jupyter_contrib_core/tree/master/tests ?